archival-python-version/progress.py

```
@@ -0,0 +1,162 @@
+    global events_to_progress, all_statuses
+
+    start_time = time.time()
+    end_time = start_time + 4  # Set to 4 seconds
+
+    statuses_to_create = [
+        InstanceStatus(str(random.randint(1, 100)), str(random.randint(1, 1000)))
+        for _ in range(task_total)
+    ]
+
+    with Live(console=console, refresh_per_second=20) as live:
+        update_table_task = asyncio.create_task(update_table(live))
```

Ruff (0.8.0)

Local variable `update_table_task` is assigned to but never used

Remove assignment to unused variable `update_table_task`

(F841)